### DISCUSSION TITLE
vue-cesium的图标icon库在哪儿呢，标签上定义完属性icon之后他是怎么取到值

### DISCUSSION THREAD

**Discussion by mengYa-create** [zouyaoji/vue-cesium#278] [General]
![9750b947df488699c0cfe393e0019cd](https://user-images.githubusercontent.com/67496448/190118347-61569395-017e-4140-9f87-2c671afd03b8.png)

这些值是怎么渲染成图标的

**Comment by zouyaoji** [comment]
其实并没有提供专门的图标库哈。只弄几个内置的，你有兴趣可以看看。
https://github.com/zouyaoji/vue-cesium/tree/dev/packages/theme-default/src/fonts

如果你要用 vc-icon 的话，它其实是从 quasar 的 q-icon 移植过来的。
icon就是常规的 font class的用法，

https://zouyaoji.top/vue-cesium/#/zh-CN/component/controls/vc-navigation 里面有些例子，如用icon: 'fa fa-compass'

还可以参考
https://www.iconfont.cn/help/detail?spm=a313x.7781069.1998910419.d8cf4382a&helptype=code
